common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -57,6 +58,20 @@ public Config getMetadata() {
         return metadata;
     }
 
+    public String getDisplayName() {
+        if (!getMetadata().key().isEmpty()) {
+            return Language.getInstance().getOrDefault(getMetadata().key() + ".name");
```

Comment:
Why not use `I18n#get`? It's simpler, and also has formatting support (not that it is needed this case).

Reply:
I can't find this `I18n`. Which package/lib is it in?

Reply:
`net.minecraft.client.resources.language` -> `I18n`

Reply:
(its basically what you are doing but shorter)